vertica-kafka-scheduler/README.md

```diff
@@ -0,0 +1,49 @@
+# Vertica-Kafka Scheduler
+
+This Docker image runs the Vertica-Kafka Scheduler.  It requires Docker or
+some compatible container environment.  
+
+## Quickstart
+
+If you're just trying to run the scheduler in a docker container, you don't
+need this repo.  Install docker and run it.
```

Comment:
This paragraph leaves me confused.  What do you need this repo for, then?  What's the "it" you run after installing docker?